doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -711,6 +714,26 @@ metadata:
   ...
 ----
 
+[[fmp-triggers-annotation]]
+==== fmp-triggers-annotation
+
+This enricher adds ImageStreamTag change triggers on Kubernetes resources that support the `image.openshift.io/triggers` annotation, such as StatefulSets, ReplicaSets and DaemonSets.
+
+The trigger is added to all containers that apply, but can be restricted to a limited set of containers using the following configuration:
+
+[source,xml]
+----
+...
+<enricher>
+    <config>
+        <fmp-triggers-annotation>
+            <containers>container-name-1,c2</containers>
```

Comment:
Yes, nice. But I think there's a limitation in the current configuration model. Enrichers use flat property configurations with non-repeated values, so that is also easy to change them from command line.